devito/propagator.py

```diff
@@ -47,11 +50,12 @@ class Propagator(object):
                            Set cache_blocking to AutoTuner instance, to use auto tuned
                            tune block sizes
     """
-    def __init__(self, name, nt, shape, stencils, factorized=None, spc_border=0,
-                 time_order=0, time_dim=None, space_dims=None, dtype=np.float32,
-                 forward=True, compiler=None, profile=False,
+    def __init__(self, name, nt, shape, stencils=None, loop_body=None, factorized=None,
```

Comment:
I am unable to understand how I would initialize a `PointData` object using pure symbolics. `PointData` loops are made using `Iteration` objects. The examples I have seen only pass them in the `time_loop_stencils` portion of the `Propagator` and not the primary kernel. What do I pass in as the primary kernel for a `Propagator` that is meant to initialize a `PointData` object?